sdk/src/wallet/bindings/nodejs/lib/Account.ts

```diff
@@ -410,6 +410,23 @@ export class Account {
         return JSON.parse(response).payload;
     }
 
+    /** Generate EVM addresses */
+    async generateEvmAddresses(
```

Comment:
This doesn't work without adding it on the Rust side to the AccountMethod enum `sdk/src/wallet/message_interface/account_method.rs`

Reply:
Ah, right.